include/deal.II/lac/scalapack.h

```diff
@@ -252,12 +252,10 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
    * The operations based on the input parameter @p transpose_B and the
    * alignment conditions are summarized in the following table:
    *
-   * | transpose_B |          Block Sizes         |                    Operation
-   * | | :---------: | :--------------------------: |
-   * :-------------------------------------------: | |   false     | $MB_A=MB_B$
-   * <br> $NB_A=NB_B$ |  $\mathbf{A} = a \mathbf{A} + b \mathbf{B}$   | |   true
-   * | $MB_A=NB_B$ <br> $NB_A=MB_B$ | $\mathbf{A} = a \mathbf{A} + b
-   * \mathbf{B}^T$  |
+   * | transpose_B |          Block Sizes         |                    Operation                  |
+   * | :---------: | :--------------------------: | :-------------------------------------------: |
+   * |   false     | $MB_A=MB_B$ <br> $NB_A=NB_B$ |  $\mathbf{A} = a \mathbf{A} + b \mathbf{B}$   |
+   * |   true      | $MB_A=NB_B$ <br> $NB_A=MB_B$ | $\mathbf{A} = a \mathbf{A} + b \mathbf{B}^T$  |
```

Comment:
@bangerth @tjhei Everything that looks like a table (containing ` | `) will not be touched.